app/src/main/kotlin/com/exallium/todoapp/allnotes/AllNotesAdapter.kt

```diff
@@ -26,6 +51,7 @@ class AllNotesAdapter(val model: AllNotesModel,
     private var dataSubscription: Subscription? = null
     private var clickSubject = PublishSubject.create<Int>()
```

Comment:
i need your help with this. clicks() returns Observable<Unit>. I'd need Observable<String>.